trc.py

```diff
@@ -118,11 +119,15 @@ def setup_wallet():
         f.close()
 
     print_info("Installing wallet dependencies...")
+    run_command("apt-get install software-properties-common -y")
     run_command("add-apt-repository ppa:bitcoin/bitcoin -y")
     run_command("apt-get update")
     run_command("apt-get --assume-yes install git unzip libboost-system-dev libboost-filesystem-dev libboost-chrono-dev "
                 "libboost-program-options-dev libboost-test-dev libboost-thread-dev software-properties-common libdb4.8-dev "
-                "libdb4.8++-dev libminiupnpc-dev libevent-dev libzmq3-dev")
+                "libdb4.8++-dev libminiupnpc-dev libevent-dev libzmq3-dev libboost-chrono1.58.0 libboost-filesystem1.58.0 "
+                "libboost-system1.58.0 libboost-thread1.58.0 libdb4.8++ libevent-2.0-5 libevent-core-2.0-5 libevent-pthreads-2.0-5 "
+                "libminiupnpc10 libsodium18 libboost-chrono1.58.0 libboost-filesystem1.58.0 libboost-system1.58.0 "
+                "libboost-thread1.58.0 libdb4.8++ libevent-2.0-5 libevent-core-2.0-5 libevent-pthreads-2.0-5 libminiupnpc10 libsodium18 libzmq5")
 
```

Comment:
Thank for adding the extra libraries. I will delete the duplicated libraries.

Reply:
This should solve the open issue.